src/iface/ethernet.rs

```diff
@@ -104,12 +106,12 @@ impl<'b, 'c, DeviceT> InterfaceBuilder<'b, 'c, DeviceT>
     /// ```
     pub fn new(device: DeviceT) -> InterfaceBuilder<'b, 'c, DeviceT> {
         InterfaceBuilder {
-            device:              device,
-            ethernet_addr:       None,
-            neighbor_cache:      None,
-            ip_addrs:            ManagedSlice::Borrowed(&mut []),
+            device:                                      device,
```

Comment:
this can be mitigated with tuning `struct_field_align_threshold` for something more sane (now 100)